app/apollo/resolvers/cluster.js

```diff
@@ -181,7 +186,13 @@ const clusterResolvers = {
           throw new NotFoundError(context.req.t('Could not find the cluster with name {{clusterName}}.', {'clusterName':clusterName}), context);
         }
 
-        await validAuth(me, org_id, ACTIONS.READ, TYPES.CLUSTER, queryName, context, [cluster.id, cluster.name, clusterName]);
+        logger.info({req_id, user, org_id, clusterName}, `${queryName} found matching cluster`);
+
+        logger.info({req_id, user, org_id, clusterName}, `${queryName} validating`);
```

Comment:
Need to reduce the number of log lines in general.  In this example, having two sequential log statements doesn't add any value

Reply:
fixed